src/contentscript/overlay/root/components/Notification/index.tsx

```diff
@@ -45,14 +46,22 @@ export const Notification: React.FC<NotificationProps> = (props) => {
   }
 
   useEffect(() => {
-    if (stateNotify && stateNotify.targetID === _id) {
+    if (
+      stateNotify &&
+      stateNotify.targetID === _id &&
+      !document
+        .querySelector('#dapplets-overlay-manager')
```

Comment:
Why it's necessary to search for the DOM selector? This doesn't seem to be how React is supposed to work. Why you can't just analyse the state? Probably, there is some another solution.